client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -270,6 +270,23 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 			}
 		}
 	}
+
+	.step-container__header .formatted-header {
+		h1.formatted-header__title {
+			font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu,
```

Comment:
You should be able to use the `$sans` Sass variable here rather than the full font stack.